src/clients/archwayd/index.js

```diff
@@ -84,65 +122,28 @@ class DockerArchwayClient extends DefaultArchwayClient {
   static #getDockerArgs(archwaydHome, archwaydVersion) {
     return [
       'run',
+      '--pull',
+      'always',
       '--rm',
       '-it',
       `--volume=${archwaydHome}:/root/.archway`,
       `archwaynetwork/archwayd:${archwaydVersion}`
     ];
   }
-
-  async checkHomePath() {
-    const oldArchwayHome = '/var/tmp/.archwayd';
-    if (this.archwaydHome === oldArchwayHome || !await pathExists(oldArchwayHome)) {
-      return;
-    }
-
-    const questions = [
-      {
-        type: 'confirm',
-        name: 'move',
-        message: chalk`I've found a keystore in {cyan ${oldArchwayHome}}. Would you like to move it to {cyan ${this.archwaydHome}}?`,
-        initial: true
-      }, {
-        type: async prev => prev && await pathExists(this.archwaydHome) ? 'confirm' : null,
-        name: 'overwrite',
-        message: chalk`The directory {cyan ${this.archwaydHome}} is not empty. Would you like to overwrite its contents?`,
-        initial: true
-      }
-    ];
-
-    try {
-      const { move, overwrite } = await prompts(questions);
-
-      if (move) {
-        await mv(oldArchwayHome, this.archwaydHome, overwrite);
-      }
-    } catch (e) {
-      if (e instanceof PromptCancelledError) {
-        console.warn(chalk`{yellow Cancelled moving keystore}`);
-      } else {
-        console.error(`Failed to move directory: ${e.message || e}\n`);
-      }
-    } finally {
-      console.info();
-    }
-  }
 }
 
-async function clientFactory({ docker = false, checkHomePath = false, ...options } = {}) {
+// TODO: remove when Docker support is removed
+async function clientFactory({ docker = false, ...options } = {}) {
   if (docker) {
-    const client = new DockerArchwayClient(options);
-    if (checkHomePath) {
-      await client.checkHomePath();
-    }
-    return client;
+    return new DockerArchwayClient(options);
   } else {
-    return new DefaultArchwayClient(options);
+    return new ArchwayClient(options);
   }
 }
 
-module.exports = {
+// TODO: export only ArchwayClient when Docker support is removed
```

Comment:
can we create an issue for those Todos? Then list everything there
Better than comments

`Clean ArchwayClient after docker support is removed` something like that and list the TODOs 
Also, add the issue number to the comment :)

Reply:
I'll list all TODOs and create proper issues